addon/data/shooter-interactive-worker.js

```diff
@@ -873,16 +887,11 @@ if (window.getSelection().rangeCount && ! window.getSelection().isCollapsed) {
   watchFunction(captureSelection)();
 }
 
-let textSelectButton;
-
 window.addEventListener("mouseup", watchFunction(function (event) {
-  if (textSelectButton) {
-    textSelectButton.parentNode.removeChild(textSelectButton);
-    textSelectButton = null;
-  }
-  if (event.target.className == "pageshot-textbutton") {
+  if (currentState !== "text") {
```

Comment:
I'd like to leave the button in.  So if currentState is text, then do captureSelection() unconditionally, otherwise still add the button.